Makefile

```diff
@@ -89,13 +89,11 @@ $(OUT)/trivialDAO_storage.tz: src/**
           ; now_val = $(now_val) \
           ; metadata_map = $(call escape_double_quote,$(metadata_map)) \
           ; ledger_lst = $(call escape_double_quote,$(ledger)) \
-          ; voting_period = $(voting_period) \
-          } \
-        ; config_data = \
-          { max_period = 2592000n \
-          ; min_period = 1n \
-          ; quorum_threshold = $(quorum_threshold) \
           } \
+          ; config_data = \
+            { voting_period = $(voting_period) \
+            ; quorum_threshold = $(quorum_threshold) \
+            } \
```

Comment:
I see you have increased the indentation, however `storage_data` and `config_data` are indeed part of the same record.

I think the correct indentation is the one that was used before (same for the other changes in this file).

Reply:
Have fixed it.

Reply:
I see. Infact I checked the other two instances originally, but somehow thought they looked alright.

I see now that the inner braces of config record does not alight with the field names above them. I believe this is what needs to be fixed and I have made that change. 

>P.s. also the review comment just above

Hope this refers to amending `docs/building.md`, in which case I have included it.